src/forms/FormLines.stories.tsx

```diff
@@ -181,6 +181,25 @@ export function WithFieldProps() {
   );
 }
 
+export function WithHorizontalLayout() {
+  return (
+    <FormLines horizontalLayout width="lg">
+      <TextField label="Name" value="first" hideLabel onChange={noop} />
+      <SelectField<Options, number>
+        label="Unit of Measure"
+        value={1}
+        hideLabel
+        options={[
+          { id: 1, name: "Each" },
+          { id: 2, name: "Square Feet" },
+        ]}
+        onSelect={noop}
+      />
+      <Switch label="Signatory" labelStyle="form" hideLabel selected={true} onChange={noop} />
```

Comment:
Would also be awesome if our `Switch` component and `RadioButtonGroup` and `CheckboxGroup` components could use the same `PresentationFieldProps` to drive the label layout.